src/index.js

```diff
@@ -1288,6 +1288,164 @@ class CloudManagerAPI {
       throw e
     })
   }
+
+  async _listContentSets (url) {
+    return this._get(url, codes.ERROR_LIST_CONTENT_SETS).then(res => {
+      return res.json()
+    }, e => {
+      throw e
+    })
+  }
+
+  async _listContentFlows (url) {
+    return this._get(url, codes.ERROR_LIST_CONTENT_FLOWS).then(res => {
+      return res.json()
+    }, e => {
+      throw e
+    })
+  }
+
+  /**
+   * Create a Content set
+   *
+   * @param {string} programId - the program id
+   * @param {string} contentSet - the body of the content set.
+   * @returns {Promise<ContentSetRepresentation>} the created content set
+   */
+  async createContentSet (programId, contentSet) {
+    const url = '/api/program/' + programId + '/contentSets'
```

Comment:
You could use string literals instead of concatenating the values.

Reply:
Done.

I pushed the changes.